forge.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  'packagerConfig': {
+    'asar': true,
+    'icon': './resources/icons/icon' // Automatically adds file extension based on OS
```

Comment:
If I understand it correctly, and forge produces EXEs, we should see to checking that the distinction 64/32/arm works on Windows